playground/src/pages/basics/CollisionDemo.vue

```diff
@@ -1,6 +1,7 @@
 <script setup lang="ts">
 import { OrbitControls } from '@tresjs/cientos'
 import { TresCanvas } from '@tresjs/core'
+// @ts-ignore
```

Comment:
Hi @Neosoulink why the ts ignore?

Reply:
You fixed the issue in your last update here -> https://github.com/Tresjs/rapier/pull/163

We were getting a TS error because we weren't receiving the type. The `ts-ignore` was there only to disable it.
But we have to remove it since we're no longer receiving any errors. Thanks to your last update!

Reply:
Ok make sense, let me know when you push @Neosoulink that should solve the lint issue